src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -288,9 +288,15 @@ def test_generate_and_write_mlbf(self):
             key = mlbf.KEY_FORMAT.format(guid=guid, version=version_str)
             assert key not in bfilter
 
-        assert os.stat(mlbf.filter_path).st_size == 203, (
+        # Occasionally a combination of salt generated with secrets.token_bytes
+        # and the version str generated in version_factory results in a
```

Comment:
Can't we set a fixed version str?

Reply:
we could but that wouldn't fix it.  The issue has more detail if the comment isn't clear.

Reply:
ha, ok. I guess we could mock the randomness to avoid the issue then?